test/templates/config/UseSecondary.py

```diff
@@ -0,0 +1,29 @@
+from WMCore.Configuration import Configuration
+config = Configuration()
+
+#General Section
+config.section_("General")
+config.General.requestName = 'CHANGE'
+config.General.workArea = 'CHANGE'
+config.General.transferOutputs = True
+config.General.transferLogs = True
+config.General.instance = 'preprod'
+
+#Job Type Section
+config.section_("JobType")
+config.JobType.pluginName = 'Analysis'
+config.JobType.psetName = 'psets/pset_parent_analysis.py'
```

Comment:
Did this get recently renamed?  I had copied the config for parent usage and changed it?  I think I might have overlooked file name changes when rebasing…